testing/bsg_dataflow/bsg_fifo_bypass/Makefile

```diff
@@ -1,5 +1,9 @@
 export BASEJUMP_STL_DIR = ../../..
 
+export BSG_CADENV_DIR = $(abspath .)/../../../../bsg_cadenv

```

Comment:
A little more flexible:

TOP ?= $(shell git rev-parse --show-toplevel)
BSG_CADENV_DIR ?= $(TOP)/../bsg_cadenv


No need for export